backend/spotihue/spotihue.py

```diff
@@ -432,7 +443,7 @@ def process_kmeans_clusters_to_light_color_values(
             X, Y, Z = self._convert_rgb_to_xyz(gamma_corrected_values)
             x, y = self._convert_xyz_to_xy(X, Y, Z)
 
-            light_color_values.append([x, y])
+            light_color_values.append((x, y))
```

Comment:
Why tuple over list? For immutability?

Reply:
I saw that this method was declared as returning a list of tuples, but technically it was returning a list of lists (not a big deal, since in our case iterating over both is the same). And yeah, I just liked the idea of updating the returned data more than updating the declared return type, I think tuples fit what we're doing.

Reply:
woops, left a response on my other account (smh)...
re-pasted:
I saw that this method was declared as returning a list of tuples, but technically it was returning a list of lists (not a big deal, since in our case iterating over both is the same). And yeah, I just liked the idea of updating the returned data more than updating the declared return type, I think tuples fit what we're doing.